website/src/pages/admin/AllDevices.tsx

```diff
@@ -33,7 +33,7 @@ export const AllDevices = observer(class AllDevices extends React.Component {
   });
 
   deleteUser = async (user: User.AsObject) => {
-    if (await confirm('Are you sure?')) {
+    if (await confirm('Delete ' + user.name + ' are you sure? ')) {
```

Comment:
'Are you sure you want to delete ' + user.name + '?'
would be a better fit

Reply:
![image](https://github.com/freifunkMUC/wg-access-server/assets/5702338/0e446ddd-693d-4fe1-84fe-69fe66f9db3e)